pax-warp-core/src/test/resources/changelogs/changelog4.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<changeLog xmlns="urn:org.ops4j.pax.warp:changelog" version="0.1">
+    <changeSet id="4">
+        <createTable tableName="foo">
+            <column name="id" type="int32" autoIncrement="true" nullable="false"/>
+        </createTable>
+        <runSql dbms="h2, oracle">INSERT INTO foo VALUES (1)</runSql>
```

Comment:
I don't think this is a good example. You shouldn't insert into autoincrement columns. This may not work on all databases.